app/services/request_approval_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RequestApprovalService < BaseService
+  def call(user)
+    return true if user.id.nil?
```

Comment:
Is the return value used for anything? I don't think so. (i.e. should simply return)